packages/apollo-cli/src/fetch-schema.ts

```diff
@@ -56,15 +51,35 @@ export async function fromFile(
 
 export const fetchSchema = async (
   { url, headers }: EndpointConfig,
-  projectFolder?: string
+  projectFolder?: string,
+  insecureEnabled?: boolean
 ): Promise<GraphQLSchema | undefined> => {
   if (!url) throw new Error("No endpoint provided when fetching schema");
   const filePath = projectFolder ? path.resolve(projectFolder, url) : url;
   if (fs.existsSync(filePath)) return fromFile(filePath);
 
+  const insecureOptionActive = insecureEnabled ? insecureEnabled : false;
```

Comment:
I think we can avoid an extra variable here by using a default parameter value (`insecureEnabled?: boolean = false`).

Reply:
Just discovered that checking a (`boolean | undefined`) variable on an if would only accept it if the variable is `true` so this line is not necessary anymore.